*** tests/A-01-06-M-ok.out	2022-10-01 08:05:11.000000000 +0100
--- tests/A-01-06-M-ok.outhyp	2022-10-19 01:06:11.803785494 +0100
***************
*** 6,12 ****
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro wqrqwrwqqw
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
--- 6,12 ----
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: wqrqwrwqqw (No such file or directory)
  MENU PRINCIPAL
  1 - Abrir
  2 - Guardar
