*** tests/A-03-03-M-ok.out	2022-10-01 17:51:24.000000000 +0100
--- tests/A-03-03-M-ok.outhyp	2022-10-19 01:06:12.031785505 +0100
***************
*** 14,20 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|0|0|0
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 14,20 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: Identificador do cliente: Visualizar cliente: Operação inválida: O cliente 'cli001' não existe.
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
