*** tests/A-11-06-M-ok.out	2022-10-30 22:15:20.000000000 +0000
--- tests/A-11-06-M-ok.outhyp	2022-11-03 00:46:15.492119465 +0000
***************
*** 63,69 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Valores para o terminal '969007':0 (pagamentos),  10 (dívidas).
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 63,69 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: 0
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
