*** tests/A-13-19-M-ok.out	2022-11-03 21:30:25.000000000 +0000
--- tests/A-13-19-M-ok.outhyp	2022-11-11 13:08:58.259766939 +0000
***************
*** 42,48 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 540 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 42,48 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 810 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
