*** tests/A-13-15-M-ok.out	2022-11-03 17:32:42.000000000 +0000
--- tests/A-13-15-M-ok.outhyp	2022-11-11 13:08:57.647778388 +0000
***************
*** 155,161 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 40 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 155,161 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
