*** tests/A-15-01-M-ok.out	2022-10-30 22:17:10.000000000 +0000
--- tests/A-15-01-M-ok.outhyp	2022-11-03 00:46:17.760130264 +0000
***************
*** 57,63 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: TEXT|1|969001|969006|10|10|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 57,63 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: TEXT|1|969001|969006|10|0|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
***************
*** 126,133 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: TEXT|1|969001|969006|10|10|FINISHED
! TEXT|2|969011|969001|49|10|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 126,133 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: TEXT|1|969001|969006|10|0|FINISHED
! TEXT|2|969011|969001|49|0|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
