*** tests/A-15-03-M-ok.out	2022-10-30 22:17:10.000000000 +0000
--- tests/A-15-03-M-ok.outhyp	2022-11-03 00:46:17.924131044 +0000
***************
*** 86,93 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: TEXT|1|969001|969007|100|200|FINISHED
! TEXT|2|969011|969014|102|204|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 86,93 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: TEXT|1|969001|969007|100|0|FINISHED
! TEXT|2|969011|969014|102|0|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
