*** tests/A-15-08-M-ok.out	2022-10-24 17:44:20.000000000 +0100
--- tests/A-15-08-M-ok.outhyp	2022-11-11 13:09:01.159712329 +0000
***************
*** 111,118 ****
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: TEXT|1|969011|969001|10|10|FINISHED
! TEXT|2|969011|969014|10|10|FINISHED
! TEXT|3|969006|969001|50|16|FINISHED
  TEXT|4|969006|969015|150|300|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
--- 111,118 ----
  7 - Mostrar terminais com saldo positivo
  0 - Sair
  Escolha uma opção: TEXT|1|969011|969001|10|10|FINISHED
! TEXT|2|969011|969014|10|5|FINISHED
! TEXT|3|969006|969001|50|8|FINISHED
  TEXT|4|969006|969015|150|300|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
