*** tests/A-18-12-M-ok.out	2022-10-30 22:20:10.000000000 +0000
--- tests/A-18-12-M-ok.outhyp	2022-11-03 00:46:20.148141620 +0000
***************
*** 137,143 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: Identificador do cliente: CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
- O2I|969013
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 137,142 ----
***************
*** 147,153 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: Identificador do cliente: CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
- O2I|969013
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 146,151 ----
