*** tests/A-20-08-M-ok.out	2022-10-29 01:04:39.000000000 +0100
--- tests/A-20-08-M-ok.outhyp	2022-11-11 13:09:07.063599387 +0000
***************
*** 72,79 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número da comunicação: O identificador da comunicação é inválido.
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 72,78 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
