*** tests/A-20-09-M-ok.out	2022-10-29 01:13:09.000000000 +0100
--- tests/A-20-09-M-ok.outhyp	2022-11-11 13:09:07.179597141 +0000
***************
*** 34,41 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número da comunicação: O identificador da comunicação é inválido.
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 34,40 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número da comunicação: OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
