*** tests/A-22-07-M-ok.out	2022-10-31 11:30:52.000000000 +0000
--- tests/A-22-07-M-ok.outhyp	2022-11-11 13:09:08.783566046 +0000
***************
*** 233,244 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: Identificador do cliente: VOICE|1|969014|969013|1|20|FINISHED
  VOICE|2|969007|969011|1|20|FINISHED
  TEXT|3|969007|969011|6|10|FINISHED
- VOICE|4|969015|969001|1|20|FINISHED
  VOICE|5|969014|969011|1|20|FINISHED
! TEXT|6|969007|969001|6|10|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 233,244 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: Identificador do cliente: VOICE|4|969015|969001|1|20|FINISHED
! TEXT|6|969007|969001|6|10|FINISHED
  VOICE|2|969007|969011|1|20|FINISHED
  TEXT|3|969007|969011|6|10|FINISHED
  VOICE|5|969014|969011|1|20|FINISHED
! VOICE|1|969014|969013|1|20|FINISHED
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
