*** tests/A-23-08-M-ok.out	2022-11-04 14:20:36.000000000 +0000
--- tests/A-23-08-M-ok.outhyp	2022-11-11 17:36:00.695513379 +0000
***************
*** 274,284 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: CLIENT|cli004|Carlos Oliveira|103449|NORMAL|YES|1|0|200
! CLIENT|cli006|Paulo Oliveira|103449|NORMAL|YES|1|0|200
! CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|1|0|40
! CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|0|40
  CLIENT|cli005|Ricardo Oliveira|103449|NORMAL|YES|1|0|40
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|1|0|20
  CONSULTAS
  1 - Mostrar todas as comunicações
--- 274,284 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: CLIENT|cli006|Paulo Oliveira|103449|NORMAL|YES|1|0|200
! CLIENT|cli004|Carlos Oliveira|103449|NORMAL|YES|1|0|200
  CLIENT|cli005|Ricardo Oliveira|103449|NORMAL|YES|1|0|40
+ CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|0|40
+ CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|1|0|40
  CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|1|0|20
  CONSULTAS
  1 - Mostrar todas as comunicações
